website/astro.config.ts

```diff
@@ -397,11 +362,10 @@ export default defineConfig({
 
 	markdown: {
 		syntaxHighlight: "prism",
-		remarkPlugins: [remarkToc],
-		rehypePlugins: [rehypeSlug, [rehypeAutolinkHeadings, autolinkConfig]],
+		rehypePlugins: [rehypeSlug, ...rehypeAutolink()],
```

Comment:
Since the `unified` project had breaking changes, we now have some weird behavior clashing issue when trying to apply the `remarkAutolink` plugin, had to move it to an external file to make it work properly.